pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/enterprise/index.md

```diff
@@ -105,8 +113,25 @@ clusterresourceset.addons.cluster.x-k8s.io/nvidia-feature-discovery-my-aws-clust
 configmap/nvidia-feature-discovery-my-aws-cluster upgraded
 ```
 
+Flags:
+    -h, --help  Help for addons
+
+Global Flags:
+    -v, --verbose int   Output verbostiy
+
+For more information about a command, use:
+
+```sh
```

Comment:
```suggestion
```bash
```

Reply:
I left it sh instead of bash because they have to change the flag so I don't think they can just copy and paste.